components/SnowLoading/index.tsx

```diff
@@ -16,11 +20,17 @@ const useStyles = makeStyles((theme) => ({
   })
 }));
 
+type SnowLoadingProps = {
+  loading: boolean,
+  height?: string | number,
```

Comment:
Avoid mixing concrete types when possible, I believe is not necessary to pass a string. Consider only using number type

Reply:
I thought we can pass both of string and number ('20px' or 20).
Okay. I will enable number only.